auto_install/install.sh

```diff
@@ -340,7 +338,7 @@ It is also possible to use a DHCP reservation, but if you are going to do that,
 setDHCPCD() {
     # Append these lines to dhcpcd.conf to enable a static IP
     echo "interface ${pivpnInterface}
-    static ip_address=${IPv4addr}
+    static ip_address=${IPv4addr}/24
```

Comment:
Hardcoding /24 subnets doesn't seem to be a correct approach.

Reply:
I think it's my fault. Basically I added that in my fork because setting a static IP on the raspberry without (my) netmask actually used a /8 subnet. So yes, perhaps there should be some code that detects the netmask or just revert to the default.